Userland/Services/WebContent/WebDriverConnection.cpp

```diff
@@ -414,7 +414,8 @@ Messages::WebDriverClient::NavigateToResponse WebDriverConnection::navigate_to(J
     m_page_client.page().load(url);
 
     // FIXME: 8. If url is special except for file and current URL and URL do not have the same absolute URL:
-    // FIXME:     a. Try to wait for navigation to complete.
+    //            a. Try to wait for navigation to complete.
+    TRY(wait_for_navigation_to_complete());
```

Comment:
Can we add the missing if statement as well?

Reply:
Sure